subprojects/kotlin-dsl-plugins/src/integTest/kotlin/org/gradle/kotlin/dsl/plugins/precompiled/PrecompiledScriptPluginAccessorsTest.kt

```diff
@@ -62,6 +62,96 @@ import java.io.File
 @LeaksFileHandles("Kotlin Compiler Daemon working directory")
 class PrecompiledScriptPluginAccessorsTest : AbstractPrecompiledScriptPluginTest() {
 
+    @Test
+    @ToBeFixedForConfigurationCache
```

Comment:
:x: Please add a `because` to the annotation

Reply:
It seems we don't add those for Kotlin DSL tests.